src/components/util/document-store.tsx

```diff
@@ -0,0 +1,28 @@
+import { getWalletDetails } from "./wallet";
+
+export const setDocumentStoreInformation = async (documentStoreAddress: string): Promise<void> => {
+  const walletDetails = await getWalletDetails();
+  if (walletDetails) {
+    const storedInformation = localStorage.getItem(`
+    ${walletDetails.network}-${walletDetails.address.toLowerCase()}`);
+
+    let documentStoreArray: string[] = [];
+    if (storedInformation != null) {
+      documentStoreArray = JSON.parse(storedInformation);
```

Comment:
let's check whether the parsed information is really an array

Reply:
i change to .split() and .join() instead of JSON.parse.